crates/cargo-test-support/src/git.rs

```diff
@@ -197,7 +197,7 @@ pub fn add_submodule<'a>(
     default_repo_cfg(&subrepo);
     t!(subrepo.remote_add_fetch("origin", "refs/heads/*:refs/heads/*"));
     let mut origin = t!(subrepo.find_remote("origin"));
-    t!(origin.fetch(&[], None, None));
+    t!(origin.fetch(&[&String::new()], None, None));
```

Comment:
Can you explain why this change is needed?  The [docs](https://github.com/libgit2/libgit2/blob/2f6f10bb804becddfae78bb9682183ae8ae8184d/include/git2/remote.h#L819-L820) still say an empty array is OK.

Reply:
Oh, I see, it is due to https://github.com/rust-lang/git2-rs/pull/482.  Perhaps it could use a cast instead?

Reply:
Yes I believe this is the cause of the broken CI as well, and agree that this should use an explicit cast and/or type parameter to convince the build it should work.